RIOT-OS/RIOT
# examples/wasm/wasm_sample: revert prebuild WASM

### Contribution description

Running `make` in the wasm example modifies the `hello_prebuild.wasm` example, making it easy to sneak in unwanted changes. This reverts such an instance and modifies the Makefile to only recreate/update `hello_prebuild.wasm` with:

    make hello_prebuild.wasm


### Testing procedure

In the PR's branch, run:

```
git diff master -- examples/wasm/wasm_sample/hello_prebuild.wasm
```

This will show that indeed this changes the file compared to master. However with

```
git diff e7d1c4abc0db3314fa9589b7119695e8959a0f49^ -- examples/wasm/wasm_sample/hello_prebuild.wasm
```

It won't differ. e7d1c4abc0db3314fa9589b7119695e8959a0f49 was the commit that sneaked in a change and this reverts it to the state before that commit (`7d1c4abc0db3314fa9589b7119695e8959a0f49^`), to which the file does no longer differ.

Additionally, running `make -C examples/wasm/wasm_sample` will no longer create `hello_prebuild.wasm`, so that no tracked files are created by a simple `make` command to avoid future changes to sneak in.

### Issues/PRs references

Change sneaked in with e7d1c4abc0db3314fa9589b7119695e8959a0f49 in https://github.com/RIOT-OS/RIOT/pull/19558
